apps/client/sttr_build/parser.ts

```diff
@@ -0,0 +1,304 @@
+import { createHash } from "https://deno.land/std/hash/mod.ts";
```

Comment:
Why is `parser.ts` with deno in this PR?

Reply:
huh?